java/samples/example.conf.properties

```diff
@@ -1,6 +1,6 @@
-client.openai.key: "OPEN_AI_KEY"
-client.openai.organizationid: "OPEN_AI_ORGANIZATION_ID"
```

Comment:
when I copy this file to `conf.properties` I still get this exception:

```
jbang sample-code/src/main/java/com/microsoft/semantickernel/samples/Example00_GettingStarted.java
Exception in thread "main" java.io.IOException: Unable to find OpenAI settings. Please provide an endpoint and organisation Id.
        at com.microsoft.semantickernel.samples.Config$ClientType.buildClient(Config.java:187)
        at com.microsoft.semantickernel.samples.Config$ClientType$1.getClient(Config.java:93)
        at com.microsoft.semantickernel.samples.Example00_GettingStarted.run(Example00_GettingStarted.java:75)
        at com.microsoft.semantickernel.samples.Example00_GettingStarted.main(Example00_GettingStarted.java:81)
       ```
       
whats the expected location or way to call it?

Reply:
found the cause - I had not specified organization id (since no other openai API I used *require* it). when specifying it things got better.

any reason why require specifying org id?

Reply:
We noticed this bug yesterday where the organization id is truly not required, but in some cases the API will not consider the setting valid.

Reply:
Check this PR for the upcoming Settings feature: https://github.com/microsoft/semantic-kernel/pull/2115